hazelcast/src/main/java/com/hazelcast/client/map/impl/nearcache/NearCachedClientMapProxy.java

```diff
@@ -503,7 +503,9 @@ public <R> R executeOnKeyInternal(Object key,
         try {
             response = super.executeOnKeyInternal(key, entryProcessor);
         } finally {
-            invalidateNearCache(key);
```

Comment:
Why don't we do a similar thing for `submitToKeysInternal` above?

Reply:
It is already added in the finally block